packages/eslint-plugin-office-addins/src/utils/utils.ts

```diff
@@ -72,12 +82,31 @@ function findOfficeApiReferencesInScope(scope: Scope): void {
   scope.childScopes.forEach(findOfficeApiReferencesInScope);
 }
 
+function isMethodFunction(node: TSESTree.MemberExpression): boolean {
+  if (node.parent?.type === AST_NODE_TYPES.CallExpression) {
+    const callExpression: TSESTree.Node = node.parent;
```

Comment:
I don't think you need a local variable for just one use in a single if statement and if you don't do that it can be a single conditional statement instead of two.  And if you are just returning a boolean that matches the result of if statement then you should just return the result of the conditional (more to the point).

If you really want to do it that way you should use the CallExpression type since that is what you just tested for